recipes/r-tidyjson/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = '0.2.2.3' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-tidyjson
+  version: {{ version|replace("-", "_") }}
+
+source:
+
+  url: https://github.com/sbilge/tidyjson/archive/v0.2.2.3.tar.gz
+  sha256: 74b29b888253983d14d612ffde09ceaa29deccb694194d51db290913a28bdfc1
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: knitr, testthat, ggplot2, magrittr
+requirements:
+  build:
```

Comment:
This recipe has no compiled code, so the only build requirement should be `{{posix}}zip` for Windows